cardinal/config.go

```diff
@@ -0,0 +1,28 @@
+package cardinal
+
+import "os"
+
+type WorldConfig struct {
+	RedisAddress       string
+	RedisPassword      string
+	CardinalWorldId    string
+	CardinalPort       string
+	CardinalDeployMode string
```

Comment:
DeployMode feels like the wrong word here, as this doesn't control any sort of "deployment", just the type of world you get. Maybe just `Mode`? not really sure here.

either way ~
the DeployMode type should be more opinionated. string is too vague.

````go
type DeployMode string

const (
    DeployModeProd =  "produdction"
    DeployModeDev = "development"
   ....
)
```

Reply:
I'm down to just call it CardinalMode and using enum